lib/workers/repository/onboarding/branch/config.ts

```diff
@@ -14,12 +14,33 @@ async function getOnboardingConfig(
 ): Promise<RenovateSharedConfig | undefined> {
   let onboardingConfig = clone(config.onboardingConfig);
 
-  let foundPreset: string | undefined;
+  const foundPreset = await searchDefaultOnboardingPreset(config.repository!);
```

Comment:
```suggestion
  // TODO #22198 fix types
  const foundPreset = await searchDefaultOnboardingPreset(config.repository!);
```
This is for `config.repository!`